inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/animation/ScaleExpandAnimator.kt

```diff
@@ -0,0 +1,85 @@
+package me.saket.inboxrecyclerview.animation
+
+import android.view.View
+import me.saket.inboxrecyclerview.InboxRecyclerView
+import me.saket.inboxrecyclerview.page.ExpandablePageLayout
+
+/**
+ * @param scaling whether to scale items or not, if set to false this animator actually
+ * looks like a simple shared element transition.
+ */
+class ScaleExpandAnimator(private val scaling: Boolean = true) : ItemExpandAnimator() {
```

Comment:
Your [screen recordings](https://imgur.com/a/mG0S94t) are very helpful in explaining the behavior when `scaling` is on/off. Let's add them to this repo and link to them here as well as in the [wiki](https://github.com/saket/InboxRecyclerView/wiki/Item-animations#2-scale).